### DISCUSSION TITLE
How to change meta description dynamically

### DISCUSSION THREAD

**Discussion by GowthamReddyAnala** [quasarframework/quasar#12388] [CLI - SSR mode]
Hi,
I want to share links through various websites like linkedin, whatsapp etc. I want to update meta data dynamically after server fetch completed
But, below I have written basic update through serverPrefetch in this case title updating but description not getting update
for production and staging we check in `https://metatags.io/`
For local builds title and description update checks in the view-page-source. anyhow here title is updating but description not getting updating in quasar 1+ it is working 
```
import { useMeta } from "quasar";
import { ref } from "vue";
export default {
	setup() {
		const title = ref("First page"); 
		const myDescription = ref({
			name: "description",
			content: "First page quasar app",
		});
		useMeta(() => {
			return {
				title: title,
				titleTemplate: (title) => `${title.value}`,
				meta: {
					description: myDescription.value,
				},
			};
		});
		const setMetaContent = () => {
			title.value = "Title Changed";
			myDescription.value = {
				name: "description",
				content: "Hey I am changed",
			};
		}
		return {
			setMetaContent,
		};
	},
	serverPrefetch() {
		this.setMetaContent();
	},
	mounted() {
		this.setMetaContent();
	},
};
```

![image](https://user-images.githubusercontent.com/22943477/152929309-8e91d948-17ef-4117-bce9-0b6e939e1b55.png)

**Comment by rstoenescu** [comment]
`title: title` is static. Should be `title: title.value`

**Reply by GowthamReddyAnala** [reply]
Hi @rstoenescu 
Here my problem is meta description not getting update

**Reply by MilosPaunovic** [reply]
[https://github.com/quasarframework/quasar/discussions/12178#discussioncomment-2046672](https://github.com/quasarframework/quasar/discussions/12178#discussioncomment-2046672)

**Reply by GowthamReddyAnala** [reply]
@MilosPaunovic 
-SSR Mode
In the above posted code snippet I am able to update meta title dynamically but not meta description

**Comment by zjhgx163** [comment]
same problem
